frontend/src/app/app-routing.module.ts

```diff
@@ -10,6 +10,7 @@ import { TasqueBoardComponent } from './components/tasque-board/tasque-board.com
 import { TasqueTeamComponent } from './components/tasque-team/tasque-team.component';
 import { TasqueProjectSettingsComponent } from './components/tasque-project-settings/tasque-project-settings.component';
 import { AuthGuard } from './auth/guards/auth.guard';
+import { NotFoundPageComponent } from 'src/shared/components/not-found-page/not-found-page.component';
```

Comment:
github actions fix. There should be a route for notFound page

Reply:
we have to fix routing first if we want to embed it into other pages